languages/kotlin/README.md

```
@@ -1,23 +1,22 @@
-Created with reference to https://github.com/bitwarden/sdk/blob/main/languages/kotlin/README.md
-
 # Android
```

Markdownlint

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)